src/servant/servant_private.h

```diff
@@ -0,0 +1,44 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+#include <servant.h>
+
+#ifndef VERSION_PSP
+// all of the shared functions need to be marked static
+static s32 AccumulateTowardZero(s32 arg0, s32 arg1);
```

Comment:
I'm thinking something like
```
#ifndef VERSION_PSP
#define STATIC static
#else
#define STATIC
#endif
```
and `STATIC void` etc. would be cleaner?

Reply:
yeah, that is way cleaner.  I'll do that.